lightning_examples/finetuning-scheduler/finetuning-scheduler.py

```diff
@@ -0,0 +1,718 @@
+# %% [markdown]
+# ## Scheduled Finetuning with the Finetuning Scheduler Extension
+#
+# ![Finetuning Scheduler logo](logo_fts.png){height="58px" width="401px"}
+#
+# The [Finetuning Scheduler](https://finetuning-scheduler.readthedocs.io/en/stable/index.html) extension accelerates and enhances model experimentation with flexible finetuning schedules.
+#
+# Training with the extension is simple and confers a host of benefits:
+#
+# - it dramatically increases finetuning flexibility
+# - expedites and facilitates exploration of model tuning dynamics
+# - enables marginal performance improvements of finetuned models
+#
+# Setup is straightforward, just install from PyPI! Since this notebook-based example requires a few additional packages (e.g.
+# ``transformers``, ``sentencepiece``), we installed the ``finetuning-scheduler`` package with the ``[examples]`` extra above.
+# Once the ``finetuning-scheduler`` package is installed, the [FinetuningScheduler](https://finetuning-scheduler.readthedocs.io/en/stable/api/finetuning_scheduler.fts.html#finetuning_scheduler.fts.FinetuningScheduler) callback is available for use with PyTorch Lightning.
+# For additional installation options, please see the Finetuning Scheduler [README](https://github.com/speediedan/finetuning-scheduler/blob/main/README.md).
```

Comment:
Great catch on the callback availability assumption. I've added clarification regarding the package installation prerequisite.